Composer/packages/tools/language-servers/language-generation/src/LGServer.ts

```diff
@@ -443,13 +444,12 @@ export class LGServer {
     const range = getRangeAtPosition(document, position);
     const wordAtCurRange = document.getText(range);
     const endWithDot = wordAtCurRange.endsWith('.');
-    const lgFile = this.getLGDocument(document)?.index();
+    const lgFile = LGParser.parseText(document.getText(), '', this.getImportResolver(document));
```

Comment:
Seems no need to change this part, the original `this.getLGDocument(document)?.index();` suppose to handle inline case. if change so, `document.getText()` may return inline body, it's not valid for `parseText`.